### ISSUE TITLE
commands on attacking machine

### ISSUE THREAD

**Issue by Gulshan45** [t0thkr1s/revshellgen#15]
Now that it starts listening on my system on 0.0.0.0 and my specified ports. Now what command should i run on target machine?

**Comment by t0thkr1s** [comment]
Hi!

It gives you the finished command as output and the script also copies that to your clipboard. You can use that output on the target machine. Hope it helps!

**Comment by Gulshan45** [comment]
should i run that command on target machine?

**Comment by t0thkr1s** [comment]
Exactly! Because that output will give you the reverse shell back to you from the target machine.

**Comment by Gulshan45** [comment]
my attacking system is ubuntu. Which would be the target system? like ubuntu, kali, windows ?

**Comment by Gulshan45** [comment]
![image](https://github.com/t0thkr1s/revshellgen/assets/55939218/0e089d0d-10b4-4629-88d8-4841c572c13b)

**Comment by Gulshan45** [comment]
since it works fine on the same network. How can we test it on different network like source machine on one network(personal hotspot) and target machine on different network(office network).?

**Comment by t0thkr1s** [comment]
you can run the command on any machine that you would like to connect back to your ubuntu. if you are in separate networks, you will need a public IP (that is accessible like a VPS) so that you can catch the reverse connection.
